src/test/java/com/diffplug/gradle/eclipse/MavenCentralMappingTest.java

```diff
@@ -42,7 +42,23 @@ private void assert463(Map<String, String> bundleToVersion) {
 		Assertions.assertThat(bundleToVersion)
 				.containsEntry("org.eclipse.debug.core", "3.10.100")
 				.containsEntry("org.eclipse.equinox.p2.metadata", "2.3.100")
-				.hasSize(895);
+				.containsEntry("org.eclipse.help", "3.7.0")
+				.hasSize(743);
+	}
+
+	@Test
+	public void testParsing4140() throws IOException, ParserConfigurationException, SAXException {
+		try (InputStream input = MavenCentralMappingTest.class.getResourceAsStream("/artifacts-4.14.0.xml")) {
+			assert4140(MavenCentralMapping.parse(input));
+		}
+	}
+
+	private void assert4140(Map<String, String> bundleToVersion) {
+		Assertions.assertThat(bundleToVersion)
+				.containsEntry("org.eclipse.debug.core", "3.14.100")
+				.containsEntry("org.eclipse.equinox.p2.metadata", "2.4.600")
+				.containsEntry("org.eclipse.help", "3.8.600")
+				.hasSize(785);
```

Comment:
`785` is also the size returned bnd when listing the content of the
`https://download.eclipse.org/eclipse/updates/4.14/`

Output: https://github.com/jmini/ecentral/blob/master/data/4.14/bnd-output.txt

---

Approach described on stackoverflow:
[How to programmatically query a p2 repository for information and artifacts?](https://stackoverflow.com/a/54445914/91497)